openapi/index.yaml

```diff
@@ -559,3 +559,8 @@ definitions:
     enum:
       - SI
       - "NO"
+  FamilyMemberCount:
+    description: Describes the count of members in a family, we also define some sane limits
+    type: integer
+    minimum: 1
+    maximum: 30
```

Comment:
I think we should have some sane limits in case we have fraud attempts or some bug on INPS, is 30 maximum family members enough?

Reply:
why do we want a maximum?

Reply:
@gunzip sanity check in case of bugs or bad data somewhere else

Reply:
Probably it's better to avoid guessing here ? :)

Reply:
What do you mean?

Reply:
I think it makes sense to have a maximum, it can be 30, 40, 50 whatever - if we get a 100 members family we should obviously reject the request :)